doc/man7/provider-digest.pod

```diff
@@ -0,0 +1,212 @@
+=pod
+
+=head1 NAME
+
+provider-digest - The digest library E<lt>-E<gt> provider functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/core_numbers.h>
+ #include <openssl/core_names.h>
```

Comment:
Because of this, you need to add the following somewhere, surrounded by empty lines:

```
=for comment multiple includes
```

Otherwise, util/find-doc-nits complains, as can be seen on Travis

Reply:
Thanks for the tip. Fixup pushed.